src/Compilers/Core/Portable/Emit/CommonPEModuleBuilder.cs

```diff
@@ -1072,7 +1083,7 @@ public ArrayMethods ArrayMethods
 
 #nullable enable
 
-        internal PrivateImplementationDetails GetPrivateImplClass(TSyntaxNode syntaxNodeOpt, DiagnosticBag diagnostics)
+        internal PrivateImplementationDetails GetPrivateImplClass(TSyntaxNode? syntaxNode, DiagnosticBag diagnostics)
```

Comment:
>TSyntaxNode? [](http://example.com/codeflow?start=66&length=12)

Given the override below, it feels like having an overload with this parameter type adds no type safety benefit.

Reply:
That isn't a big problem I guess.